doc/nrf/releases/release-notes-changelog.rst

```diff
@@ -305,6 +305,8 @@ Libraries for networking
   * Added:
 
     * A public header file :file:`nrf_cloud_defs.h` that contains common defines for interacting with nRF Cloud and the :ref:`lib_nrf_cloud` library.
+    * A new event :c:enum:`NRF_CLOUD_EVT_TRANSPORT_CONNECT_ERROR` to indicate an error while the transport connection is being established when the :kconfig:option:`CONFIG_NRF_CLOUD_CONNECTION_POLL_THREAD` Kconfig option is enabled.
+      Earlier this was indicated with a second :c:enum:`NRF_CLOUD_EVT_TRANSPORT_CONNECTING` event with an error status.
```

Comment:
```suggestion
      Earlier, this was indicated with an :c:enum:`NRF_CLOUD_EVT_TRANSPORT_CONNECTING` event with an error status.
```
Does this go wrong if we don't say second?

Reply:
I think we want to say it's the second `NRF_CLOUD_EVT_TRANSPORT_CONNECTING` event. The first one is sent always, but in case of an error, a second `NRF_CLOUD_EVT_TRANSPORT_CONNECTING` event is sent with a different status. This was not very intuitive (and not really documented).